eclipse-collections-code-generator/src/main/resources/api/map/mutablePrimitiveObjectMap.stg

```diff
@@ -45,20 +53,71 @@ public interface Mutable<name>ObjectMap\<V> extends <name>ObjectMap\<V>, Mutable
     }
 
     /**
+     * Puts all of the key/value mappings from the specified map into this map. If this
+     * map already has a value associated with one of the keys in the map, it will be
+     * replaced with the value in {@code map}.
+     * @param map the map to copy into this map
      * @since 5.0.
      */
     void putAll(<name>ObjectMap\<? extends V> map);
 
+    /**
+     * Removes the mapping associated with the key, if one exists, from the map.
+     * @param key the key to remove
+     * @see #remove(<type>)
+     */
     V removeKey(<type> key);
 
+    /**
+     * Removes the mapping associated with the key, if one exists, from the map.
+     * @param key the key to remove
+     * @see #removeKey(<type>)
+     */
     V remove(<type> key);
 
+    /**
+     * Retrieves the value associated with the key if one exists; if it does not,
+     * associates a value with the key.
+     * a new value with they key
+     * @param key the key
+     * @param value the value to associate with {@code key} if no such mapping exists
+     * @return the value associated with key, if one exists, or {@code value} if not
+     */
     V getIfAbsentPut(<type> key, V value);
 
+    /**
+     * Retrieves the value associated with the key if one exists; if it does not,
+     * associates the result of invoking the value supplier with the key.
```

Comment:
I don't believe the key is passed to the function, as it is a `Function0` which means it takes no paramters.

Reply:
That is what I meant, but I agree it's a bit confusing. By saying "value supplier" I was hoping that readers would realize a `Function0` == `Supplier<>` from Java 8.  I was trying to say "associates ... with the key" not "associates ... invoking ... with the key". I'll try to word it more clearly.